GeneralsMD/Code/GameEngineDevice/CMakeLists.txt

```diff
@@ -194,6 +194,20 @@ set(GAMEENGINEDEVICE_SRC
     Source/Win32Device/GameClient/Win32Mouse.cpp
 )
 
+# Add C++ 17 FileSystem implementation for non-VS6 builds
+if(NOT IS_VS6_BUILD)
```

Comment:
> Question is if i should enable this for the VS2022 build

I suggest VS2022 uses the legacy implementation. Reason being, our first releases are meant to be shipped as VS6 builds, so we need our VS2022 setup close to that for testing.